### DISCUSSION TITLE
Facets -> Can you control capitalization

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#15209] [Q&A] [answered]
Facets are a very nice feature but is it possible to control how it capitalizes the range output?

We produce facets on a set of raven document identifiers, however, when you do facets on them they are lowercase. Is there a config item that prevents this?

e.g The document identifier is operators/0000000000000394935-A but when we do a facet on them it produces operators/0000000000000394935-a

**Comment by ayende** [comment, marked as answer]
The facet value is the "analyzed" term in the index.
You can use `Exact` model in the index, and it will give you the unmodified term, yes.
